### PR TITLE
Add features module for organization switch repo

### PR DESCRIPTION
**Pull request by sadilchamishka**

### CODE REVIEW

File: features/etc/feature.properties [hunk 1]
```diff
@@ -0,0 +1,200 @@
+#
+# Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (AnuradhaSK) [inline features/etc/feature.properties:2]: All Rights Reserved part is not available in public source code license

File: features/etc/feature.properties [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (AnuradhaSK) [inline features/etc/feature.properties:4]:
```suggestion
# WSO2 Inc. licenses this file to you under the Apache License,
```
Inc -> LLC 

Please check and add the correct license

File: features/etc/feature.properties (hunk 1 above)
Comment (sadilchamishka) [reply features/etc/feature.properties]: Fixed by - https://github.com/wso2-extensions/identity-oauth2-grant-organization-switch/pull/16/commits/0b171a59a8c453736a4336a50e5c4d33325006c6

File: features/etc/feature.properties (hunk 2 above)
Comment (sadilchamishka) [reply features/etc/feature.properties]: Fixed by - https://github.com/wso2-extensions/identity-oauth2-grant-organization-switch/pull/16/commits/0b171a59a8c453736a4336a50e5c4d33325006c6

**Review by AnuradhaSK — APPROVED** [review]

**Pull request merged**